airflow/ui/src/components/DataTable/searchParams.ts

```diff
@@ -17,12 +17,14 @@
  * under the License.
  */
 import type { SortingState } from "@tanstack/react-table";
-
 import type { TableState } from "./types";
+import { SearchParams } from "src/constants/searchParams";
 
-export const LIMIT_PARAM = "limit";
-export const OFFSET_PARAM = "offset";
-export const SORT_PARAM = "sort";
+const {
+  LIMIT: LIMIT_PARAM,
+  OFFSET: OFFSET_PARAM,
+  SORT: SORT_PARAM
+} = SearchParams;
```

Comment:
CamelCasing is for React components and Types.
`searchParams` would conflict with the actual values too.
How about `searchParamKeys`?

Reply:
I think the convention for enum is CamelCasing but i can renamed `SearchParams` into `SearchParamsKeys`